tests/kernel/install_kotd.pm

```diff
@@ -109,11 +109,19 @@ sub run {
 
     my $baseurl = "http://download.suse.de/ibs/Devel:";
     my $version = "SLE" . get_required_var("VERSION");
+    my $subdir  = "standard";
     if (is_opensuse) {
         $baseurl = "http://download.opensuse.org/repositories";
         $version = "HEAD";
+        # currently only x86_64, i686 and i586 are supported in standard directory
+        if (get_required_var('ARCH') =~ /aarch|arm/) {
```

Comment:
I haven´t found any medium type with arch arm. I think it´s safe to do

    if (get_required_var('ARCH') =~ /aarch64/) {

Reply:
I thought it does not harm to address all possibilities (why not being ready), but I can remove it.

Reply:
I think it's actually better to use common utils for openqa:

https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/Utils/Architectures.pm

    use Utils::Architectures;

    $subdir = "ARM" if is_aarch64();
    $subdir = "PPC" if is_ppc64le();

Reply:
Good point, updated.